src/components/ProfileButton/MainProfileButton.jsx

```diff
@@ -0,0 +1,27 @@
+import clsx from "clsx";
+
+import mainDefaultIcon from "../../assets/icons/person.svg";
+
+import styles from "./MainProfileButton.module.css";
+
+const MainProfileButton = ({ src = mainDefaultIcon, className, onClick }) => (
+  <button
+    className={clsx(styles.mainProfileButton, className)}
+    onClick={onClick}
+  >
```

Comment:
버튼에 대한 다른 속성도 전달받을 수 있도록 rest 파라미터를 전달해도 좋을 것 같아요 😊
```suggestion
const MainProfileButton = ({ src = mainDefaultIcon, className, onClick, ...props }) => (
  <button
    className={clsx(styles.mainProfileButton, className)}
    onClick={onClick}
    {...props}
  >
```

Reply:
SOLID
I: (Interface Segregation Principle)

꼭 모든 경우의 수를 열어놓는 것이 항상 정답 아니라는 것도 고민해주시면 좋을 것 같아요